test.sh

```diff
@@ -464,6 +507,13 @@ test_tickets_endpoint()
 
 test_users_groups_endpoint()
 {
+    curl -H "authorization: Bearer $bearer_token" "${base_url}/users-groups" \
+        --data-urlencode 'op=modify_metadata' \
+        --data-urlencode 'name=kory' \
```

Comment:
let's remove kory from the test

Reply:
This file may go away entirely since the official test suite is written in python.

Reply:
Created issue #86 to address this.